scripts/install-freetype

```diff
@@ -0,0 +1,28 @@
+#CLEAN IF ANY
+rm -rf $(grep -Ril "freetype" /opt/devkitpro/portlibs/nds/)
+
+#INSTALL
+pkgver=$1
+
+wget http://download.savannah.gnu.org/releases/freetype/freetype-$pkgver.tar.gz
+tar -xvf freetype-$pkgver.tar.gz
+
+cd freetype-$pkgver
+source /opt/devkitpro/ndsvars.sh
+./configure --prefix="${PORTLIBS_PREFIX}" --host=arm-none-eabi \
```

Comment:
We are not meant to install into PORTLIBS_PREFIX, that's owned by devkitPro dkp-pacman.